### PR TITLE
adding signalslot recipe

### PR DESCRIPTION
**Pull request by kbeeperez**
implements: 
https://github.com/nsls-ii-forge/signalslot-feedstock
https://github.com/nsls-ii-forge/weakrefmethod-feedstock

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by kbeeperez** [comment]
Attn @mrakitin

File: recipes/weakrefmethod/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - weakrefmethod
+
+about:
+  home: "http://github.com/twang817/weakrefmethod"
+  license: "Python Software Foundation"
```
Comment (mrakitin) [inline recipes/weakrefmethod/meta.yaml:30]:
```suggestion
  license: PSF-2.0
```

File: recipes/weakrefmethod/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 37bc1fbb5575acf82172d4eb7b6fc4412d77d5a1d70dff2c1f8a4574301cda66
+
+build:
+  number: 0
+  noarch: "python"
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (mrakitin) [inline recipes/weakrefmethod/meta.yaml:15]:
```suggestion
  noarch: python
  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

File: recipes/weakrefmethod/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{% set name = "weakrefmethod" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
```
Comment (mrakitin) [inline recipes/weakrefmethod/meta.yaml:6]:
```suggestion
  name: {{ name|lower }}
  version: {{ version }}
```

File: recipes/weakrefmethod/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```
Comment (mrakitin) [inline recipes/weakrefmethod/meta.yaml:9]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipes/weakrefmethod/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - weakrefmethod
+
+about:
+  home: "http://github.com/twang817/weakrefmethod"
```
Comment (mrakitin) [inline recipes/weakrefmethod/meta.yaml:29]:
```suggestion
  home: https://github.com/twang817/weakrefmethod
```

File: recipes/signalslot/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,46 @@
+{% set name = "signalslot" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```
Comment (mrakitin) [inline recipes/signalslot/meta.yaml:9]:
```suggestion
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipes/signalslot/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+  sha256: 676e913cd6aefb8ef5f5ef368cc85bd8bc8847a12fb00348f2bdfe78abb0e3de
+
+build:
+  number: 0
+  noarch: "python"
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv "
```
Comment (mrakitin) [inline recipes/signalslot/meta.yaml:15]:
```suggestion
  noarch: python
  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

File: recipes/signalslot/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+
```
Comment (mrakitin) [inline recipes/signalslot/meta.yaml:21]:
```suggestion
```

File: recipes/signalslot/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+    - signalslot
+    - signalslot.contrib
+    - signalslot.contrib.task
+
+about:
+  home: "https://github.com/numergy/signalslot"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: LICENSE
+  summary: "Simple Signal/Slot implementation"
```
Comment (mrakitin) [inline recipes/signalslot/meta.yaml:40]:
```suggestion
  home: https://github.com/numergy/signalslot
  license: MIT
  license_family: MIT
  license_file: LICENSE
  summary: "Simple Signal/Slot implementation"
```

**Review by mrakitin — APPROVED** [review]
Added a few formatting suggestions.

@conda-forge/staged-recipes, I agree to be listed as a maintainer for these 2 recipes.

**Review by mrakitin — APPROVED** [review]
@kbeeperez, thanks for addressing the suggestions.
@dopplershift, ready for review/merge!

**Review by dopplershift — APPROVED** [review]

**Comment by mrakitin** [comment]
Thanks, @dopplershift!

**Pull request merged**